src/IISIntegration/test/Common.FunctionalTests/Inprocess/FeatureCollectionTests.cs

```diff
@@ -25,5 +26,13 @@ public async Task FeatureCollectionTest_SetHttpContextFeatures(string path)
         {
             Assert.Equal("Success", await _fixture.Client.GetStringAsync(path + "/path" + "?query"));
         }
+
+        [ConditionalFact]
+        [RequiresNewHandler]
+        [RequiresNewShim]
```

Comment:
*New*Handler, *New*Shim? Does that mean v2, or is it dynamic/latest?

Reply:
Latest.